grails-app/assets/javascripts/index/IndexControl.js

```diff
@@ -0,0 +1,216 @@
+const form = document.getElementById("form")
+const username = document.getElementById("username")
+const email = document.getElementById("email")
+const cpfcnpj = document.getElementById("cpfcnpj")
+const telefone = document.getElementById("telefone")
+const cep = document.getElementById("cep")
+const endereco = document.getElementById("endereco")
+const numero = document.getElementById("numero")
+const bairro = document.getElementById("bairro")
+const cidade = document.getElementById("cidade")
+const estado = document.getElementById("estado")
+const password = document.getElementById("password")
+const confirmpassword = document.getElementById("confirmpassword")
+
+
+
+
+
+
+form.addEventListener("submit", (event) => {
+    event.preventDefault();
+    checkInputs();
+    if(checkInputs()===true){
+        alert("Enviado com sucesso!!")
+    }else{
+        alert("Preencha os campos em branco")
+    }
+    var lista_dados = [username.value, email.value, cpfcnpj.value, telefone.value, cep.value, 
+        endereco.value, numero.value, bairro.value, cidade.value, estado.value, password.value, confirmpassword.value];
+    console.log(lista_dados)    
+    
+    document.getElementById("username").value = "";
+    document.getElementById("email").value = "";
+    document.getElementById("cpfcnpj").value = "";
+    document.getElementById("telefone").value = "";
+    document.getElementById("cep").value = "";
+    document.getElementById("endereco").value = "";
+    document.getElementById("numero").value = "";
+    document.getElementById("bairro").value = "";
+    document.getElementById("cidade").value = "";
+    document.getElementById("estado").value = "";
+    document.getElementById("password").value = "";
+    document.getElementById("confirmpassword").value = "";  
+    
+
+});
+
+
+
+
+
+function checkInputs(){
+    const usernameValue = username.value.trim()
+    const emailValue = email.value.trim()
+    const cpfcnpjValue = cpfcnpj.value.trim()
+    const telefoneValue = telefone.value.trim()
+    const cepValue = cep.value.trim()
+    const enderecoValue = endereco.value.trim()
+    const numeroValue = numero.value.trim()
+    const bairroValue = bairro.value.trim()
+    const cidadeValue = cidade.value.trim()
+    const estadoValue = estado.value.trim()
+    const passwordValue = password.value.trim()
+    const confirmpasswordValue = confirmpassword.value.trim()
+
+    var usernameOk, emailOk, cpfcnpjOk, telefoneOk, cepOk, enderecoOk, numeroOk, bairroOk, cidadeOk, estadoOk, passwordOk, confirmpasswordOk
+
+
+    if(usernameValue === ""){
```

Comment:
a negação do valor vazio funciona 
```suggestion
    if(!usernameValue){
```